pkg/block/block_linux.go

```diff
@@ -210,6 +210,13 @@ func diskPartitions(ctx *context.Context, paths *linuxpath.Paths, disk string) [
 }
 
 func diskPartUUID(ctx *context.Context, part string) string {
+	if !ctx.EnableTools {
+		// intentionally not using ctx.Warn because
+		// 1. we want to avoid excessive message flow
+		// 2. this is an explicit setting, which client code
+		//    must be prepared to handle.
+		return ""
+	}
```

Comment:
I guess I'm not entirely sure why we're retunrning `""` here and not outputting at least a warning message, since this is changing the behaviour of `ghw` without any indication to the user. We warn below on original line 224 and 234 if we encounter an error using the tool `blkid` so I'm unsure why this isn't similar?

Reply:
good point. This is a plain old mistake on my side. Will revisit and re-upload.